extensions/pixabay/src/lib/api.ts

```diff
@@ -8,6 +8,13 @@ import fetch from "cross-fetch";
 
 const streamPipeline = util.promisify(pipeline);
 
+export function getClipboardDownloadFolder(): string {
+  const d = "~/Downloads/Pixabay";
```

Comment:
Is there a specific reason why you don't want to store into the downloads folder for the clipboard?

Reply:
@tonka3000 Nope, switched back to ~/Downloads.